src/components/Chart/LineChart.jsx

```diff
@@ -150,7 +161,13 @@ class LineChart extends Component {
 }
 
 LineChart.propTypes = {
-  width: PropTypes.number.isRequired,
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      x: PropTypes.any,
+      y: PropTypes.any
```

Comment:
👍 

PropTypes.number for x and y

Reply:
Actually it can also be a `Date` (like in our current case). I don't know if we can really enforce more than just a shape `{ x, y }`